schema/cron.sh

```diff
@@ -30,19 +30,19 @@ if [ ! -z "$existing_branch" ]; then
 fi
 
 # Generate enigma-go based on latest published Qlik Associative Engine image
-./schema/generate.sh
+. ./schema/generate.sh
 
 # If there are changes to qix_generated.go then open a pull request
 local_changes=$(git ls-files qix_generated.go -m)
 
 if [ ! -z "$local_changes" ]; then
   git checkout -b $branch_name
   git add qix_generated.go
-  git commit -m "Automated: Generated enigma-go based on new JSON-RPC API"
+  git commit -m "Automated: Generated enigma-go based on new JSON-RPC API for version $ENGINE_VERSION"
```

Comment:
It's a bit verbose :) Maybe just `Automated: New API based on $ENGINE_VERSION`

Reply:
Fixed